iceoryx_hoofs/test/moduletests/test_concurrent_sofi.cpp

```diff
@@ -275,7 +275,7 @@ TEST_F(CUnitTestContainerSoFi, SoFiSizeEqualsNumberOfPushes)
     for (uint32_t i = 0; i < TEST_SOFI_CAPACITY; i++)
     {
         EXPECT_EQ(m_sofi.size(), i);
-        m_sofi.push(i, ret);
+        m_sofi.push(static_cast<int>(i), ret);
```

Comment:
OK for this PR but this was and is shaky without considering context: depends on `TEST_SOFI_CAPACITY` which is likely small enough, < 2^31) and whether it is OK for the remaining test logic to push negative numbers.

May break in the future if the context changes.